packages/p2p-v2/types.ts

```diff
@@ -104,3 +107,5 @@ export type TStep = {
     };
     subStepCount?: number;
 };
+
+export type TCountryListItem = NonNullable<THooks.Country.Get>;
```

Comment:
I think we can remove the NonNullable since above its already wrapped with it, can you check

```suggestion
export type TCountryListItem = THooks.Country.Get;
```